sdk/translation/azure-ai-translation-text/src/main/java/com/azure/ai/translation/text/TextTranslationAsyncClient.java

```diff
@@ -862,28 +684,95 @@ public Mono<List<BreakSentenceItem>> findSentenceBoundaries(List<InputTextItem>
+     * @param body Defines the content of the request.
+     * @param clientTraceId A client-generated GUID to uniquely identify the request.
+     * @throws IllegalArgumentException thrown if parameters fail the validation.
+     * @throws HttpResponseException thrown if the request is rejected by server.
+     * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
+     * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
+     * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return the response.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<List<TransliteratedText>> transliterate(String language, String sourceLanguageScript,
+        String targetLanguageScript, List<String> body, String clientTraceId) {
+        return transliterateInner(language, sourceLanguageScript, targetLanguageScript, convertTextToData(body),
+            clientTraceId);
+    }
+
+    /**
+     * Transliterate Text.
+     * <p>
+     * This method is used when you have single text to transliterate and you want to provide client trace id.
+     * </p>
+     *
+     * @param language Specifies the language of the text to convert from one script to another.
+     * Possible languages are listed in the transliteration scope obtained by querying the service
+     * for its supported languages.
+     * @param sourceLanguageScript Specifies the script used by the input text. Look up supported languages using the
+     * transliteration scope,
+     * to find input scripts available for the selected language.
+     * @param targetLanguageScript Specifies the output script. Look up supported languages using the transliteration
+     * scope, to find output
+     * scripts available for the selected combination of input language and input script.
+     * @param text Defines the content of the request.
+     * @param clientTraceId A client-generated GUID to uniquely identify the request.
+     * @throws IllegalArgumentException thrown if parameters fail the validation.
+     * @throws HttpResponseException thrown if the request is rejected by server.
+     * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
+     * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
+     * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return the response.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<List<TransliteratedText>> transliterate(String language, String sourceLanguageScript,
```

Comment:
Same here - single text, single target language should just return a `Mono<TransliteratedText>`

Reply:
Done